modules/07 NodeJS/Assignments/11/nodejs11/server/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "main.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.17.1"
+  }
+}
```

Comment:
the player doesn't load new videos...

Reply:
It does load. Its because i copied the same url for all videos.